### DISCUSSION TITLE
vector zonal stats: default param request

### DISCUSSION THREAD

**Discussion by mosesckim** [thinkingmachines/geowrangler#62] [Enhancements Requests]
Noticed default is to fill NaNs with zero after aggregation; this might make it difficult to identify original NaNs if there are actual zeros in the aggregation.

https://github.com/thinkingmachines/geowrangler/blob/master/geowrangler/vector_zonal_stats.py#L205

Also, a suggestion would be, in the case `fillna` option is set to True, make the replacement value (0 in this case at the moment) a variable users can input (e.g. -1, etc.)

Thanks!
Moses

**Comment by jtmiclat** [comment]
This makes sense. Created an issue https://github.com/thinkingmachines/geowrangler/issues/64 to track the task!

**Reply by ghost** [reply]
Locking this conversation as we have already closed issue #64.
